build/android/bytecode/java/org/brave/bytecode/BraveNewTabPageLayoutClassAdapter.java

```diff
@@ -19,6 +19,9 @@ public BraveNewTabPageLayoutClassAdapter(ClassVisitor visitor) {
         deleteField(sBraveNewTabPageLayoutClassName, "mMvTilesContainerLayout");
         makeProtectedField(sNewTabPageLayoutClassName, "mMvTilesContainerLayout");
 
+        deleteField(sBraveNewTabPageLayoutClassName, "mLogoCoordinator");
```

Comment:
Please add tests for each bytecode change

Reply:
Yes forgot that. Added now.